lib/arc_ecto/schema.ex

```diff
@@ -2,6 +2,26 @@ defmodule Arc.Ecto.Schema do
   defmacro __using__(_) do
     quote do
       import Arc.Ecto.Schema
+
+      def delete_attachments(changeset_or_data, fields) do
+        scope = case changeset_or_data do
+          %Ecto.Changeset{} -> Ecto.Changeset.apply_changes(changeset_or_data)
```

Comment:
This scenario works for removing attachment with parent entity. But what if we will add the third case?
When field has changes we could remove attachment from S3 because field would be updated with new attachment without removing parent entity.